arho_feature_template/gui/dialogs/plan_regulation_group_form.py

```diff
@@ -85,6 +85,7 @@ def __init__(self, regulation_group: RegulationGroup):
 
         self.type_of_regulation_group.populate_from_code_layer(PlanRegulationGroupTypeLayer)
         self.type_of_regulation_group.removeItem(0)  # Remove NULL from selections
+        self.type_of_regulation_group.removeItem(0)  # Remove Yleismääräykset from selections
```

Comment:
Itemien järjestys voi periaatteessa joskus muuttua, jolloin nämä ei enää ole 0 ja 1 indekseissä.
Olisi kaiketi parempi implisiittisesti poistaa joko datan tai tekstin perusteella.
`CodeComboBox`iin voisi toteuttaa esim metodin:
```python
    def remove_item_by_text(self, text: str):
        index = self.findText(text)
        if index != -1:
            self.removeItem(index)
```
Jolloin nämä saisi selkeämpään muotoon:
```suggestion
        self.type_of_regulation_group.remove_item_by_text("NULL")
        self.type_of_regulation_group.remove_item_by_text("Yleismääräykset")
```